scripts/main.py

```diff
@@ -301,36 +301,45 @@ def parse_arguments():
     except Exception as e:
         print_to_console("Error: \n", Fore.RED, str(e))
 
-    if not cfg.continuous_mode:
+    if not cfg.continuous_mode or next_action_count > 0:
```

Comment:
```suggestion
    if not cfg.continuous_mode and next_action_count === 0:
```
due to line :308 and last of any else: statement. After this change, line :308 can be removed